content/docs/hooks-faq.md

```diff
@@ -439,13 +438,13 @@ function ScrollView({row}) {
 }
 ```
 
-This might look strange at first, but an update during rendering is exactly what `getDerivedStateFromProps` has always been like conceptually.
+Hal ini bisa jadi terlihat aneh pada awalnya, tetapi sebuah pembaruan selama proses *render* adalah hal yang persis secara konsep sejak `getDerivedStateFromProps` dibuat.
 
-### Is there something like forceUpdate? {#is-there-something-like-forceupdate}
+### Apakah ada suatu hal seperti *forceUpdate*? {#is-there-something-like-forceupdate}
```

Comment:
```suggestion
### Apakah ada fungsionalitas seperti *forceUpdate*? {#is-there-something-like-forceupdate}
```

Reply:
>### Apakah ada fungsionalitas seperti *forceUpdate*? {#is-there-something-like-forceupdate}

Suggested change's applied.
Table of contents also updated